app/src/main/java/org/simple/clinic/scanid/ui/TextWatcherAdapter.kt

```diff
@@ -0,0 +1,18 @@
+package org.simple.clinic.scanid.ui
+
+import android.text.Editable
+import android.text.TextWatcher
+
+abstract class TextWatcherAdapter : TextWatcher {
```

Comment:
Since this is not specifically related to the Scan BP Passport feature, should we move it to a util package?